sale_order_import/models/sale_import_default.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2023 Quartile Limited
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class SaleImportDefault(models.Model):
+    _name = "sale.import.default"
```

Comment:
```suggestion
    _description = "Sale Import Default"
```
You should get a warning in the log without this.